### DISCUSSION TITLE
Cannot authenticate tumblr

### DISCUSSION THREAD

**Discussion by myohmyoutlook** [TumblThreeApp/TumblThree#476] [General]
Since the update today, I am no longer able to authenticate tumblr.

I can select authenticate and see my feed in the windows that appears, but doing so no longer shows that I am logged in under Settings like it used to do.

I also get an error when downloading saying I need to be logged in, even though I am logged in and have authenticated. It's like the program no longer detects that I am logged in. I even tried logging out of tumblr and logging back in but Settings is still not seeing my login.

Twitter login still works, but tumblr no longer works.

Thanks.

**Comment by thomas694** [comment]
Try deleting the `Cookies.json` in the settings (%LOCALAPPDATA%\TumblThree\Settings) folder to see if that helps.

**Reply by olegsh23** [reply]
> я удалил, мне не помогло, до этого все роботала

translated:
I deleted it, it didn’t help me, before that everything worked

**Reply by Linktydraa** [reply]
@thomas694 

I recently had tumblethree start logging me out consistently as well. It'd only stay logged in for maybe 10-15 minutes max. I have deleted the cookies file. I have cleared the cookie cache in Edge and deleted the settings files. Restarted Tumblthree, even my computer. Nothing works. I keep having to reauthenticate every so often. Waay too frequently.

**Reply by Linktydraa** [reply]
@thomas694 

Tuimblthree now also seems to just simply refuse to authenticate on tumblr. The x/twitter authentication works just fine, but no matter how many time sI authentice for tumblr, it never actually goes through. When the browser window comes up, it shows me logged in, but no matter how many times I save it, or even leave the browser window open, the authentication never completes and says signed in.

**Reply by Linktydraa** [reply]
@thomas694 

I've deleted cookies, cleared out the Edge and Explorer cookies, restarted both the app and computer, and it refuses to authenticate tumblr. I've had the window open for hours, waiting for it to authenticate, and it never does. When I lose the window, after a few seconds the button goes from grey, back to it normal state, wanting me to authenticate. It's an endless cycle of not working right.

**Reply by Linktydraa** [reply]
@thomas694 

???

**Reply by thomas694** [reply]
This error occurs for many users, but with very different levels of intensity and in different constellations, which makes it difficult to reproduce reliably. Because of that, we still don’t have enough consistent data to isolate the cause. Your report is logged with the others, and once we have a confirmed fix or workaround, we’ll post it here.

By the way, clearing all browser data or rebooting the whole PC is rarely necessary. In most cases, restarting the app is enough. For testing purposes, deleting the `Cookies.json` file is usually the only additional step that helps.

**Comment by Linktydraa** [comment]
Hi Thomas,I can’t begin to tell you how many times I’ve deleted the cookies.json file and settings files haha. That was the first thing I did in the long list of attempted troubleshooting I tried. That sucks that not only is it happening for so many other users, but each case seem unique enough to make finding the solution so difficult. Warm regards,Preston Martin On Mar 4, 2026, at 7:53 PM, Thomas ***@***.***> wrote:﻿
This error occurs for many users, but with very different levels of intensity and in different constellations, which makes it difficult to reproduce reliably. Because of that, we still don’t have enough consistent data to isolate the cause. Your report is logged with the others, and once we have a confirmed fix or workaround, we’ll post it here.
By the way, clearing all browser data or rebooting the whole PC is rarely necessary. In most cases, restarting the app is enough. For testing purposes, deleting the Cookies.json file is usually the only additional step that helps.

**Comment by Linktydraa** [comment]
Hi Thomas,

I saw you sent an update, but it doesn't seem to include anything about
tumblthree not authenticating tumblr.

On Wed, Mar 4, 2026 at 7:53 PM Thomas ***@***.***> wrote:

> This error occurs for many users, but with very different levels of
> intensity and in different constellations, which makes it difficult to
> reproduce reliably. Because of that, we still don’t have enough consistent
> data to isolate the cause. Your report is logged with the others, and once
> we have a confirmed fix or workaround, we’ll post it here.
>
> By the way, clearing all browser data or rebooting the whole PC is rarely
> necessary. In most cases, restarting the app is enough. For testing
> purposes, deleting the Cookies.json file is usually the only additional
> step that helps.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TumblThreeApp/TumblThree/discussions/476#discussioncomment-16003685>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEPBKJ3ADBLCRJQ5BTYW6CT4PDFYNAVCNFSM6AAAAACRWCLA7KVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTMMBQGM3DQNI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


-- 
Warm regards,

**Reply by thomas694** [reply]
We’d love to deliver a fix - tiny problem: one doesn’t exist yet. As soon as it does, we’ll be thrilled to include it.

**Comment by Linktydraa** [comment]
Hahah. That’s fair. Warm regards,Preston Martin On Mar 29, 2026, at 5:19 PM, Thomas ***@***.***> wrote:﻿
We’d love to deliver a fix - tiny problem: one doesn’t exist yet. As soon as it does, we’ll be thrilled to include it.

**Comment by Someonemustnothavethis** [comment]
My monthly scrape of the 196 blogs I like (don't ask) went perfectly again this morning,.  It started already logged in, updated, then went about it's merry way quietly grabbing 15 GB ( 21,900 files) of naughty stuff.  It took about 5 hours.  
It may help that I treat my install of T3 very nicely.  I take her out to dinner once a week and buy her nice things.  Flowers help too.  :-)
